### PR TITLE
Release 4.4.0

### PR DESCRIPTION
**Pull request by lcharette**

### CODE REVIEW

File: app/sprinkles/core/src/Bakery/DebugCommand.php [hunk 1]
```diff
@@ -43,42 +43,58 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
+        if (version_compare($phpVersion, \UserFrosting\PHP_MIN_VERSION, '<')) {
             $this->io->error('UserFrosting requires php version ' . \UserFrosting\PHP_MIN_VERSION . " or above. You'll need to update you PHP version before you can continue.");
             exit(1);
         }
 
         // Check for deprecated versions
-        if (version_compare(phpversion(), \UserFrosting\PHP_RECOMMENDED_VERSION, '<')) {
-            $this->io->warning('While your PHP version is still supported by UserFrosting, we recommends version ' . \UserFrosting\PHP_RECOMMENDED_VERSION . ' or above as ' . phpversion() . ' will soon be unsupported. See http://php.net/supported-versions.php for more info.');
+        if (version_compare($phpVersion, \UserFrosting\PHP_RECOMMENDED_VERSION, '<')) {
+            $this->io->warning('While your PHP version is still supported by UserFrosting, we recommends version ' . \UserFrosting\PHP_RECOMMENDED_VERSION . ' or above as ' . $phpVersion . ' will soon be unsupported. See http://php.net/supported-versions.php for more info.');
```
[the remark this answers is not in the document]
Comment (Silic0nS0ldier) [reply app/sprinkles/core/src/Bakery/DebugCommand.php]: Nope, still has security support for 8 more months.

File: app/sprinkles/account/src/Controller/AccountController.php [hunk 2]
```diff
@@ -97,7 +94,7 @@ public function checkUsername(Request $request, Response $response, $args)
         /** @var \UserFrosting\Sprinkle\Core\Util\ClassMapper $classMapper */
         $classMapper = $this->ci->classMapper;
 
-        /** @var \UserFrosting\I18n\MessageTranslator $translator */
+        /** @var \UserFrosting\I18n\Translator $translator */
```
[the remark this answers is not in the document]
Comment (lcharette) [reply app/sprinkles/account/src/Controller/AccountController.php]: Yup! Done in the [braking change section](https://github.com/userfrosting/learn/blob/develop/pages/21.upgrading/05.43-to-44/docs.md#major-likely-to-break-your-project). Will also add to upgrade portion, fair point.

File: app/sprinkles/core/composer.json [hunk 3]
```diff
@@ -39,15 +40,15 @@
...
+        "slim/slim": "^3.12",
         "slim/twig-view": "^2.5",
         "symfony/http-foundation": "*",
         "twig/twig": "^2.11",
         "userfrosting/assets": "^6.0.0",
         "userfrosting/config": "~4.3.0",
         "userfrosting/cache": "~4.3.0",
-        "userfrosting/fortress": "~4.3.0",
-        "userfrosting/i18n": "~4.3.0",
+        "userfrosting/fortress": "dev-develop",
```
[the remark this answers is not in the document]
Comment (lcharette) [reply app/sprinkles/core/composer.json]: Yup! on my list ;)

File: app/sprinkles/admin/tests/Integration/Controller/GroupControllerTest.php [hunk 4]
```diff
@@ -675,19 +675,19 @@ public function testUpdateInfoWithDuplicateName(GroupController $controller)
...
+    private function setupUser(): void
     {
         // Admin user, WILL have access
-        $testUser = $this->createTestUser(true, true);
+        $this->createTestUser(true, true);
 
         // Create test role
         $fm = $this->ci->factory;
-        $role = $fm->create('UserFrosting\Sprinkle\Account\Database\Models\Group', [
+        $fm->create('UserFrosting\Sprinkle\Account\Database\Models\Group', [
```
[the remark this answers is not in the document]
Comment (lcharette) [reply app/sprinkles/admin/tests/Integration/Controller/GroupControllerTest.php]: Yeah, will probably need to be done throughout the code.

**Pull request merged**